demos/simple/errors/verify.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+
+cargo clean
+
+# this should detect multiple errors
+( cargo-verify --tests --verbose > out || true )
+cat out
+
+# check for all expected errors
+grep -q "test assert_eq_should_fail ... ASSERT_FAILED" out
```

Comment:
add `-F` to interpret the string as fixed text instead of the default regular expression.

Reply:
cool - done